data/cleansed/payment-methods.json

```diff
@@ -374,6 +374,21 @@
     ],
     "capabilities": []
   },
+  {
+    "id": "grabpay_SG",
+    "type": "online",
+    "name": "GrabPay",
+    "small_width": 40,
+    "small_height": 26,
+    "medium_width": 77,
+    "medium_height": 50,
+    "large_width": 154,
+    "large_height": 100,
```

Comment:
All the payment method logos have the same width and height defined here, but many of them don't match the actual width and height of the logo. I guess the dimensions are not used anywhere so it's not a problem. But I still set the actual width and height of the logo here.
Let me know if I missed anything.

Reply:
I have spoken to @RoryH on this and he found that the only place this is used internally re managedmarkets is testbench - it always takes the largest image and scales it to 50px wide.

If shopify are pulling these images the mismatch in image size actual vs configured likely does not matter to them - but i think it's correct to have them match and be consistent.